napari_svg/layer_to_xml.py

```diff
@@ -248,7 +246,7 @@ def shapes_to_xml(data, meta):
         edge_width = np.ones(len(data))
 
     if 'opacity' in meta:
-        opacity = meta['opacity']
+        opacity = meta['opacity'] * np.ones(len(data))
```

Comment:
Rather than make this a vector that get's iterated in the zip do you just want to change the else statement to be 

```python
  else:
     opacity = 1
```

 then drop `opacity` from the zip and change https://github.com/sofroniewn/napari-svg/blob/834f7d104065618ae8a71ed78c399379b4404078/napari_svg/layer_to_xml.py#L270 to be
```python
props = {'stroke-width': str(ew), 'opacity': str(opacity)}
```

Reply:
makes sense to me!